code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -128,6 +133,7 @@
 	drink_icon = "mojito"
 	drink_name = "Glass of Mojito"
 	drink_desc = "Fresh from Spesscuba."
+	taste_message = "mojito"
```

Comment:
As its a mixed drink, could maybe have its own taste message.

Going to brain dump potential variants for you, ignore/use as you desire:
"an alcoholic breath mint"
"happiness"
"holiday"